run_basic_scenario.sh

```diff
@@ -1,70 +1,70 @@
 #!/usr/bin/env bash
 
-set -euo pipefail
+set -xeuo pipefail
 
 cd -- "$(dirname -- "${BASH_SOURCE[0]}")"
 
 # Deploy test canister
-
 ./deploy_test_canister.sh
+
 # assert the default greeting text
 [ "$(./bin/dfx canister call test greet "M")" == '("Hoi, M!")' ] && echo "OK" || exit 1
 
-# Deploy SNS
+# Add NNS Root as a co-controller of the dapp canisters to be decentralized.
+./let_nns_control_dapp.sh
 
-./deploy_sns.sh sns-test.yml
-# assert the SNS swap lifecycle
-[ "$(./get_sns_swap_state.sh | sed "s/0 : float32/0 : nat64/" | ./bin/idl2json | jq -r '.swap[0].lifecycle')" == "1" ] && echo "OK" || exit 1
-# assert that all SNS canister IDs have been returned by SNS-W
+# Create Service Nervous System.
+./propose_sns.sh
+
+# Assert that all SNS canister IDs have been returned by SNS-W
 jq -r '.governance_canister_id' -e sns_canister_ids.json
 jq -r '.index_canister_id' -e sns_canister_ids.json
 jq -r '.ledger_canister_id' -e sns_canister_ids.json
 jq -r '.root_canister_id' -e sns_canister_ids.json
 jq -r '.swap_canister_id' -e sns_canister_ids.json
 
-# Register test canister
+# Assert the SNS swap lifecycle is in the OPEN state.
+[ "$(./get_sns_swap_state.sh | sed "s/0 : float32/0 : nat64/" | ./bin/idl2json | jq -r '.swap[0].lifecycle')" == "2" ] && echo "OK" || exit 1
 
-./register_dapp.sh "$(./bin/dfx canister id test)"
-./wait_for_last_sns_proposal.sh
-# assert that the test canister is indeed registered
-[ "$(./get_sns_canisters.sh  | ./bin/idl2json | jq -r '.dapps[0]')" == "$(./bin/dfx canister id test)" ] && echo "OK" || exit 1
-
-# Upgrade test canister
+# Assert that the test canister is indeed registered.
+[ "$(./get_sns_canisters.sh | ./bin/idl2json | jq -r '.dapps[0]')" == "$(./bin/dfx canister id test)" ] && echo "OK" || exit 1
 
+# Upgrade test canister (I)
 ./upgrade_test_canister.sh Hello
 ./wait_for_last_sns_proposal.sh
 ./wait_for_canister_running.sh "$(./bin/dfx canister id test)"
+
 # assert the new greeting text
 [ "$(./bin/dfx canister call test greet "M")" == '("Hello, M!")' ] && echo "OK" || exit 1
 
-# Open SNS swap
-
-./open_sns_swap.sh
 ./wait_for_last_nns_proposal.sh
```

Comment:
Could you please add a comment about this line? Is the NNS proposal to create the SNS supposed to be executed only after the test canister upgrade setting the greeting message to "Hello"?

Reply:
I think this is just a leftover from the previous runbook. I'm removing this line. 

Please let me know if you disagree.